catalog/utilities/media_props_gen/docs/media_props.md

```diff
@@ -2,424 +2,376 @@
-
-## Existing Data Inconsistencies
+[`expire_old_images`](https://github.com/WordPress/openverse/blob/main/catalog/dags/retired/common/loader/sql.py)
+DAG added in
+[Expiration of outdated images in the database](https://github.com/cc-archive/cccatalog/pull/483)
+was used to set `removed_from_source` to `True` for images that were updated
+more than `OLDEST_PER_PROVIDER` value.
 
 # filetype
 
 ## Description
 
+The filetype of the media item (not the MIME type). If the filetype is not
+available in the API response, it can be extracted from the URL extension or
+from the HEAD response from the media direct URL.
+
 ## Object Shape
 
-## Selection Criteria
+String, the file type (extension) of the audio file. Maximum length: 5
+characters.
 
 ## Normalization and Validation
 
-## Existing Data Inconsistencies
+`catalog/dags/common/extensions.py` The `get_file_extension` function is used to
+get the file extension from a URL. The function returns the file extension in
+lowercase. Equivalent image file types are normalized to a single file type, see
+`FILETYPE_EQUIVALENTS`.
 
 # category
 
 ## Description
 
-## Object Shape
+One of the media category Enum values:
+[`ImageCategory`](https://github.com/WordPress/openverse/blob/48e300d10286506454a8bea17d46740dbffc5c87/catalog/dags/common/loader/provider_details.py#L137-L141)
+and
+[`AudioCategory`](https://github.com/WordPress/openverse/blob/48e300d10286506454a8bea17d46740dbffc5c87/catalog/dags/common/loader/provider_details.py#L144-L151).
 
 ## Selection Criteria
 
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+Category is assigned heuristically based on the extension and default categories
+per provider.
 
 # standardized_popularity
 
 ## Description
 
-## Object Shape
-
-## Selection Criteria
+Normalized popularity, a calculated column. Only available for providers that
+have popularity data.
 
 ## Normalization and Validation
 
-## Existing Data Inconsistencies
+The value is updated monthly during the data refresh process.
 
 # duration
 
 ## Description
 
-## Object Shape
-
-## Selection Criteria
-
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+The duration of the main audio file in milliseconds.
 
 # bit_rate
 
 ## Description
 
-## Object Shape
-
-## Selection Criteria
-
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+The bit rate of the main audio file.
 
 # sample_rate
 
 ## Description
 
-## Object Shape
-
-## Selection Criteria
-
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+The sample rate of the main audio file.
 
 # genres
 
 ## Description
 
-## Object Shape
-
-## Selection Criteria
-
-## Normalization and Validation
-
-## Existing Data Inconsistencies
+List of genres associated with the audio.
 
 # alt_files
 
 ## Description
 
+The list of alternative file details for the audio (different formats/ quality).
+
 ## Object Shape
 
-## Selection Criteria
+JSONb array of dictionaries:
```

Comment:
Nit: this is a third, not-internally-consistent way of saying the column type for JSONB columns 😅